package.json

```diff
@@ -32,6 +32,8 @@
     "ns-upload": "./bin/ns-upload.sh",
     "ns-upload-entries": "./bin/ns-upload-entries.sh",
     "oref0": "./bin/oref0.sh",
+    "oref0-autotune-test": "./bin/oref0-autotune-test.sh",
```

Comment:
I'm renaming and adding this as oref0-ns-autotune in my `autotune-install` branch.  Want to just leave this off of yours, and I'll get it on mine, to avoid merge conflicts?